guides/v2.2/config-guide/bootstrap/magento-modes.md

```diff
@@ -47,8 +47,12 @@ You can run Magento in any of the following *modes*:
 	</tr>
 	<tr>
 		<td>production</td>
-		<td><p>Intended for deployment on a production system. Exceptions are not displayed to the user, exceptions are written to logs only, and static view files are served from cache only. New or updated files are not written to the file system.</p>
-			<p>In production mode, you cannot enable or disable cache types using the Magento Admin. <a href="{{ page.baseurl }}config-guide/cli/config-cli-subcommands-cache.html#config-cli-subcommands-cache-en">More information about enabling and disabling the cache</a>.</p></td>
+		<td>Intended for deployment on a production system. This mode:
```

Comment:
Rewrite to maintain consistency with other mode descriptions:

>_Intended for deployment on a production system, this mode:_

Also, wrap the first sentence in <p> tags to create proper line spacing.

>`<p>Intended for deployment on a production system. This mode:</p>`